index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>fastify - Fast and low overhead web framework, for Node.js</title>
+  <meta name="description" content="Super fast, all natural JSON logger for Node.js">
+  <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+  <link rel="stylesheet" href="//unpkg.com/docsify/lib/themes/vue.css">
+</head>
+<body>
+  <div id="app"></div>
+</body>
+<script>
+  window.$docsify = {
+    name: 'pino',
+    loadSidebar: true,
```

Comment:
The documentation says this can be a string path to the sidebar file. How about `/docsify/sidebar.md` instead?